crypto/src/main/scala/org/bitcoins/crypto/DLEQUtil.scala

```diff
@@ -81,20 +78,19 @@ object DLEQUtil {
     // if R = y*R' which is what we are trying to prove.
     val s = fe.multiply(e).add(k)
 
-    (s, e)
+    (e, s)
   }
 
   /** Verifies a proof that the DLOG_G of P1 equals the DLOG_adaptor of P2 */
   def dleqVerify(
```

Comment:
I believe this is the correct reference for a scaladoc? 

https://github.com/discreetlogcontracts/dlcspecs/blob/d01595b70269d4204b05510d19bba6a4f4fcff23/ECDSA-adaptor.md#verifying

Reply:
I'm having a hard time understanding the parameter mapping from our implementation and what the spec says here

We have paraemter `s`, `e`, `p1`, `adaptor`, `p2` and the spec has `X`,`Y`,`Z` (points) and then a `proof` 

What is the mapping between our parameters and the spec parameters? Can we rename our parameters to be consistent with the spec so this is easier to review?